_pytest/config.py

```diff
@@ -942,6 +941,7 @@ def _consider_importhook(self, args, entrypoint_name):
         and find all the installed plugins to mark them for re-writing
         by the importhook.
         """
+        import pkg_resources
         ns, unknown_args = self._parser.parse_known_and_unknown_args(args)
         mode = ns.assertmode
         if mode == 'rewrite':
```

Comment:
@nicoddemus do we have a potential behaviour bug wrt plugin loading there

it seems to work as intended it shouldnt be in the else branch below